testutils/helper/curl.go

```diff
@@ -26,11 +28,9 @@ type CurlOpts struct {
 	WithoutStats bool
 }
 
-func (t *testContainer) CurlEventuallyShouldRespond(opts CurlOpts, substr string, ginkgoOffset int, timeout ...time.Duration) {
+func (t *testContainer) getTimeouts(timeout ...time.Duration) (currentTimeout time.Duration, pollingInterval time.Duration) {
```

Comment:
this method appears to have nothing to do with `testContainer`, want to remove the receiver? also how is this to be used?

Reply:
I will remove the reciever, also it was a part of the function which needed to be copied, so I put it all in one function. Yuval wrote it originally